leftwm-core/src/state.rs

```diff
@@ -23,7 +23,7 @@ pub struct State {
     pub mode: Mode,
     pub layouts: Vec<Layout>,
     pub scratchpads: Vec<ScratchPad>,
-    pub active_scratchpads: HashMap<String, Option<u32>>,
+    pub active_scratchpads: HashMap<String, VecDeque<u32>>,
```

Comment:
Would it be possible to create some new-types instead of using `String` and `u32`? Something like:
```rust
pub type TagName = String;
pub type ScratchPadId = u32;
// ...
pub active_scratchpads: HashMap<TagName, VecDeque<ScratchpadId>>,
```
(I don't know if those names are right but it would help to understand this attribute faster)